k6script.js

```diff
@@ -0,0 +1,37 @@
+import { check, sleep } from "k6";
+import { Rate } from "k6/metrics";
+import http from "k6/http";
+
+// See https://docs.k6.io/docs/options for other options
+export let options = {
+  // simulate rampup of traffic from 1 to 200 users over 5 minutes.
+  stages: [
+    { duration : "1m", target : 200},
+    { duration : "2m", target: 4000},
+    { duration : "1m", target : 5500},
+    { duration : "2m", target : 3000},
+    { duration : "1m", target : 1000}
+  ]
+};
+
+let errorRate = new Rate("API errors");
+
+let getUser = function(){
+    let min = 1;
+    let max = 9999999;
+    var rand = Math.floor(Math.random() * (max - min)) + min;
+    let baseUrl = 'http://localhost:3001';
+  let listingRes = http.get(`${baseUrl}/rooms/${rand}/`, {}, {
```

Comment:
To reduce the amount of code here, you could alternatively add in `baseURL` in `listingRes` instead of setting  a separate variable for it.